docker-compose.yml

```
@@ -0,0 +1,28 @@
+version: '3'
+
+services:
```

```suggestion
services:
```
バージョン表記はおそらく必要ないので削除お願いします！